ui-core/src/styles/inputs/checkbox.css

```diff
@@ -17,7 +17,7 @@
   .label {
     line-height: 1.5rem;
     padding-top: 0.188rem;
-    width: 12.813rem;
+    padding-right: 0.3rem;
```

Comment:
```suggestion
    padding-inline: 0.188rem 0.3rem;
```

Reply:
We are setting the padding top and right, not left and right

Reply:
Oh boy, my eyes aren't fixed yet, sorry about that